legate/driver/driver.py

```diff
@@ -226,35 +224,36 @@ def print_verbose(
 
     Returns
     -------
-        None
+        str
 
     """
+    out = StringIO()
 
-    print(f"\n{rule('Legion Python Configuration')}")
+    out.write(f"\n{rule('Legion Python Configuration')}")
```

Comment:
`out.write` doesn't add the newline automatically, as `print` does.

Reply:
doh https://github.com/nv-legate/legate.core/pull/899/commits/9b97a36ef085fe0cd6c087f46caad89a979d8a4f